src/coreclr/gc/objecthandle.cpp

```diff
@@ -66,7 +66,7 @@ void CALLBACK VariableTraceDispatcher(_UNCHECKED_OBJECTREF *pObjRef, uintptr_t *
     }
 }
 
-#if defined(FEATURE_COMINTEROP) || defined(FEATURE_COMWRAPPERS) || defined(FEATURE_OBJCMARSHAL) || defined(FEATURE_NATIVEAOT)
```

Comment:
This deletes number of `FEATURE_NATIVEAOT` ifdefs in the GC. `FEATURE_NATIVEAOT` ifdefs in the GC are anti-patern.